server/views/pages/start/legal-sender-start-page.njk

```diff
@@ -51,12 +51,7 @@
             }) }}
 
             <h2 class="govuk-heading-m">Prisons you can send mail to</h2>
-            <p>You can use this service to send legal or confidential mail to the following prisons:</p>
-            <ul class="govuk-list govuk-list--bullet" data-qa="prisonNames">
-            {% for prisonName in prisonNames %}
```

Comment:
`prisonName` data (and the API call behind it) is still getting passed to this template even though you're not displaying it. See here:https://github.com/ministryofjustice/send-legal-mail-to-prisons/blob/fe0388dec1495eb476609de6e741eaaac945eb73/server/routes/start/StartPageController.ts#L12

You could get rid of most of the lines in that file as you just need to render the template; no need to do API call. (Might then need to tidy up a bit as `prisonService` would no longer need to be passed)

Reply:
Good spot, thanks. Updated StartPageController.ts to remove the call to PrisonService as advised.